src/index.html

```diff
@@ -7,6 +7,320 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+
+    <header class="header">
+      <a href="#home" class="logo">
+        <img src="styles/images/logo.png" alt="Moyo logo">
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#apple" class="nav__link is-active">Apple</a>
+          </li>
+          <li class="nav__item">
+            <a href="#samsung" class="nav__link">Samsung</a>
+          </li>
+          <li class="nav__item">
+            <a href="#smartphones" class="nav__link">Smartphones</a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#laptops&computers"
+              class="nav__link"
+              data-qa="nav-hover"
+            >
+              Laptops & computers
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#gadgets" class="nav__link">Gadgets</a>
+          </li>
+          <li class="nav__item">
+            <a href="#tablets" class="nav__link">Tablets</a>
+          </li>
+          <li class="nav__item">
+            <a href="#photo" class="nav__link">Photo</a>
+          </li>
+          <li class="nav__item">
+            <a href="#video" class="nav__link">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="container">
+      <div class="container_item">
+        <div class="card" data-qa="card">
+          <img
+            class="card__img"
+            src="styles/images/imac.jpeg"
+            alt=" Macbook logo"
+          >
+          <div class="card__content">
+            <div class="card__title">
+              APPLE A1419 iMac 27" Retina
+              5K Monoblock (MNED2UA/A)
+            </div>
+          </div>
+          <p class="card__article">Product code: 195434</p>
+          <div class="stars">
```

Comment:
`stars-[modifier]` should be added

Reply:
I dont know.how fix tests fails

Reply:
still not fixed

Reply:
please add the modifier `stars-4` and proper styles in CSS

Reply:
still not fixed. it should be solution from [stars](https://github.com/mate-academy/layout_stars) task